tests/e2e/test_office_hours_queue.py

```diff
@@ -288,7 +293,7 @@ def expectedAlerts(self, success=0, error=0, success_text=[], error_text=[]):
         self.assertEqual(self.countAlertError(error_text), error)
 
 def enableQueue(self):
-    self.log_in(user_id='instructor')
+    #self.log_in(user_id='instructor')
```

Comment:
I think this line should just be removed, I dont see a reason to keep it in the code but commented out.

Reply:
oh yeah, i just forgot to remove it